graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/rest/PipelineResource.java

```diff
@@ -84,21 +96,31 @@ public class PipelineResource extends RestResource implements PluginRestResource
             .put(PipelineDao.FIELD_TITLE, SearchQueryField.create(PipelineDao.FIELD_TITLE))
             .put(PipelineDao.FIELD_DESCRIPTION, SearchQueryField.create(PipelineDao.FIELD_DESCRIPTION))
             .build();
+    public static final String GL_INPUT_ROUTING_PIPELINE = "gl_input_routing_pipeline";
```

Comment:
I don't know if [this](https://graylogdocumentation.atlassian.net/wiki/spaces/ECT/pages/3551068182/Input+Setup+Wizard#Not-using-Illuminate-content-workflow%3A) is up to date but according to this text:

> A new default, immutable pipeline is attached to the “All Messages” Stream, called the “All Messages Routing” pipeline. 

we should name the pipeline “All Messages Routing”

Reply:
Fixed